runtime/src/dpe_crypto.rs

```diff
@@ -281,21 +281,31 @@ impl<'a> Crypto for DpeCrypto<'a> {
         match algs {
             AlgLen::Bit256 => Err(CryptoError::Size),
             AlgLen::Bit384 => {
+                // derive an EC key pair from the CDI
+                let (_, pub_key) = Self::derive_key_pair(self, algs, cdi, label, info)?;
+
+                // use the entropy from the EC public key to derive an HMAC key
+                let mut hmac_key = Array4x12::default();
+                let pub_key_x: [u8; 48] = pub_key
+                    .x
+                    .bytes()
+                    .try_into()
+                    .map_err(|_| CryptoError::Size)?;
```

Comment:
I think I would prefer to compute the sha384 hash of the whole public key and then use that output as the hmac key. The reason is that the distribution of a public key isn't completely random, its selected from a discrete set of points. Hashing it should ensure they key has the pull 384 bits of entropy.

Reply:
Can we use a [KDF](https://github.com/chipsalliance/caliptra-sw/blob/main/drivers/src/hmac384_kdf.rs)? A bit more NIST-friendly if we do it that way.

Reply:
As discussed in the meeting, I changed it to hash the pub key and feed that to the hmac kdf as ikm. We can revisit kdf-ing this hash later.

Reply:
sha384 of the entire generated public key and then using that output directly as hmac-sha384 key is simplest and cryptographically fine wrt to bit munging and uniformity of the output.

A KDF step is more NIST-friendly but we already lost them on the entropy->private-key->public-key munging in the prior steps imo. And labels and different output lengths are not relevant for this single fixed sized output either.

I prefer simpler here. More comments in the code about the reasons for this particular work-around are probably nice for posterity.